src/cozy-ui.d.ts

```diff
@@ -15,7 +15,7 @@ declare module 'cozy-ui/transpiled/react/Divider' {
 
 declare module 'cozy-ui/transpiled/react/ListItem' {
   import ListItem from '@material-ui/core/ListItem'
-  export default ListItem
+  export default ListItem as ListItem & { ellipsis: boolean }
```

Comment:
Wondering if we should keep track of all types we should fix on cozy-ui and cozy-client somewhere?

Reply:
Very good question but I don’t have clear view where to put it. Maybe should we focus on adding jsdoc on cozy-ui dans generate type there 🤔